docs/reference/tools-reference/open_source_llm_tool.md

```diff
@@ -0,0 +1,59 @@
+
+This Prompt flow supports two different LLM API types:
+
+- **Completion**: to generate text based on provided prompts.
+- **Chat**: to facilitate interactive conversations with text-based inputs and responses.
+
+## Prerequisite
+
+1. Pick the model to use with from the [Azure Machine Learning model catalog](https://ml.azure.com/model/catalog).
+2. Use the "Deploy" button to deploy the model to AzureML Inference endpoint.
+
+More detailed instructions can be found here [Deploying foundation models to endpoints for inferencing.](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-use-foundation-models?view=azureml-api-2#deploying-foundation-models-to-endpoints-for-inferencing)
+
+## **Connections**
+
+Setup connections to provisioned resources in prompt flow.
+
+| Type        | Name     | API KEY  | API Type | API Version |
+|-------------|----------|----------|----------|-------------|
+| CustomConnection | Required | Required | -        | -           |
+
+Instructions to create a Custom Connection [can be found here.](https://learn.microsoft.com/en-us/azure/machine-learning/prompt-flow/how-to-integrate-with-langchain?view=azureml-api-2#create-a-connection)
+
+The keys to set are:
+
+1. endpoint_url
+2. endpoint_api_key
+3. model_family
+
+*These values can be found at the previously created Inferencing endpoint.*
+
+## Inputs
+
+| Name                   | Type        | Description                                                                             | Required |
+|------------------------|-------------|-----------------------------------------------------------------------------------------|----------|
+| api | string | this will be Completion or Chat, depending on the scenario selected | Yes |
+| connection | CustomConnection | the name of the connection which points to the Inferencing endpoint | Yes |
+| model_kwargs | dictionary | generic model configuration values, for example temperature | Yes |
+| deployment_name | string | the name of the deployment to target on the MIR endpoint. If no value is passed, the MIR load balancer settings will be used. | No |
+| prompt | string | text prompt that the language model will complete | Yes |
+
+## Outputs
+
+| API        | Return Type | Description                              |
+|------------|-------------|------------------------------------------|
+| Completion | string      | The text of one predicted completion     |
+| Chat       | string      | The text of one response of conversation |
+
+## How to use Open Source LLM Tool?
+
+1. Choose a Model from the catalog and deploy.
+2. Setup and select the connections to model deployment.
+3. Configure the model api and its parameters
+4. Prepare the Prompt with [guidance](./prompt-tool.md#how-to-write-prompt).
```

Comment:
This hyperlink is invalid.

Reply:
Copied from the llm-tool.md.
I can remove the ./ in the hope it doesn't try to use the current URL during the Pull Request - but I believe on the docs page it would be correct.

Reply:
Didn't work - at the end of the day it's a relative link.